development-updates.md

```diff
@@ -95,6 +97,12 @@ Development updates from [`doulos819`](https://github.com/doulos819)
 ### Update 1
 [Link to Update #1](https://github.com/doulos819/mjr/blob/main/research/EPF/Weekly%20Updates/Week%200.md)
 
+#Crypdough.eth
+Development updates from [`Crypdough.eth`](https://github.com/crypdoughdoteth)
+
+### Update 1
+[Update #1](https://hackmd.io/eFND-6qnTn6fhZKTlBMr4g?view)
```

Comment:
Looks like access to the document is forbidden:
![image](https://user-images.githubusercontent.com/6136245/198849523-1dc9e0c3-c4da-4729-a1b9-28275b84bcf2.png)

You can click in the top right `Share`, set `Read = Everyone`, and save.

Reply:
Thank you for pointing this out, I just fixed the link